src/app/main/footer/footer.component.html

```diff
@@ -21,13 +21,13 @@
         <div class="box contact">
             <a mat-button href="https://dasch.swiss" target="_blank">Data and Service Center for the Humanities</a>
             <a mat-button href="https://www.google.com/maps/place/Bernoullistrasse+32%2C+4056+Basel" target="_blank">
```

Comment:
google maps link still points to Bernoullistrasse

Reply:
@RalfBarkow thanks. I have corrected the links.